### PR TITLE
Develop/neonatal/haemolytic/data inputs

### PR DESCRIPTION
**Pull request by beatrixkh**
Added data inputs tables. Still need to find birth prevalence.

### CODE REVIEW

File: docs/source/models/gbd2017/causes/neonatal_jaundice/index.rst [hunk 1]
```diff
@@ -25,3 +27,104 @@ for GBD are EHB from `Rhesus (Rh) disease`_, preterm birth,
...
+	  - Measure
+	  - Value
+	  - Notes
+	* - C
+	  - prevalence
+	  - prev_c384
+	  -
+	* - C
+	  - birth prevalence
+	  - birth_prev_unknown
```
Comment (NathanielBlairStahn) [inline docs/source/models/gbd2017/causes/neonatal_jaundice/index.rst]: I think this should just be birth_prevalence_c384, and then in the data sources table, we explain how to get it. Or, if we can't directly pull birth prevalence (e.g. if we need to pull count data and divide by births), put that calculation here, and explain how to pull the values we need in the data sources table.

File: docs/source/models/gbd2017/causes/neonatal_jaundice/index.rst [hunk 2]
```diff
@@ -25,3 +27,104 @@ for GBD are EHB from `Rhesus (Rh) disease`_, preterm birth,
...
+	  - is this the table we want to house this in?
+	* - C
+	  - emr
```
Comment (NathanielBlairStahn) [inline docs/source/models/gbd2017/causes/neonatal_jaundice/index.rst]: I think we should write out the full name: excess mortality rate

File: docs/source/models/gbd2017/causes/neonatal_jaundice/index.rst [hunk 3]
```diff
@@ -25,3 +27,104 @@ for GBD are EHB from `Rhesus (Rh) disease`_, preterm birth,
...
+	* - F
+	  - emr
+	  - 0
+	  -
+	* - F
+	  - disability weight
+	  - 0
+	  -
+	* - All
+	  - CSMR
```
Comment (NathanielBlairStahn) [inline docs/source/models/gbd2017/causes/neonatal_jaundice/index.rst]: Again, full name = cause-specific mortality rate

File: docs/source/models/gbd2017/causes/neonatal_jaundice/index.rst [hunk 4]
```diff
@@ -25,3 +27,104 @@ for GBD are EHB from `Rhesus (Rh) disease`_, preterm birth,
...
+	
+	* - Variable
+	  - Source
+	  - Notes
+	* - prev_c384
+	  - como
+	  - 
+	* - birth_prev_unknown
+	  - como?
+	  - have asked helena about this
```
Comment (collijk) [inline docs/source/models/gbd2017/causes/neonatal_jaundice/index.rst]: Birth prevalence is pulled from como.  It uses the incidence measure id and a special age group id that I can't remember off the top of my head.

File: docs/source/models/gbd2017/causes/neonatal_jaundice/index.rst [hunk 5]
```diff
@@ -25,3 +27,104 @@ for GBD are EHB from `Rhesus (Rh) disease`_, preterm birth,
...
+	:widths: 10 10 20
+	:header-rows: 1
+	
+	* - Variable
+	  - Source
+	  - Notes
+	* - prev_c384
+	  - como
+	  - 
+	* - birth_prev_unknown
```
Comment (NathanielBlairStahn) [inline docs/source/models/gbd2017/causes/neonatal_jaundice/index.rst]: If there is a value we can pull directly, we should call it birth_prevalence_c384, and explain how to get it in the notes column. If we have to combine multiple raw values to get the right number, we need to list each of those in this table.

File: docs/source/models/gbd2017/causes/neonatal_jaundice/index.rst [hunk 6]
```diff
@@ -25,3 +27,115 @@ for GBD are EHB from `Rhesus (Rh) disease`_, preterm birth,
...
+	* - F
+	  - prevalence
+	  - 1-prevalence_c384
+	  -
+	* - F
+	  - birth prevalence
+	  - 1-birth_prevalence_c384
+	  - 
+	* - F
+	  - emr
```
Comment (NathanielBlairStahn) [inline docs/source/models/gbd2017/causes/neonatal_jaundice/index.rst]:
```suggestion
	  - excess mortality rate
```

File: docs/source/models/gbd2017/causes/neonatal_jaundice/index.rst [hunk 7]
```diff
@@ -25,3 +27,115 @@ for GBD are EHB from `Rhesus (Rh) disease`_, preterm birth,
...
+	  - Description
+	  - Notes
+	* - prevalence_c384
+	  - como
+	  - Prevalence of hemolytic disease and other neonatal jaundice
+	  - 
+	* - birth_prevalence_c384
+	  - como
+	  - Proportion of babies born with hemolytic disease and other neonatal jaundice
+	  - age_group_id = 164 and measure = 6 (incidence)
```
Comment (NathanielBlairStahn) [inline docs/source/models/gbd2017/causes/neonatal_jaundice/index.rst]: Does this get returned as a rate or a count?

File: docs/source/models/gbd2017/causes/neonatal_jaundice/index.rst [hunk 8]
```diff
@@ -25,3 +27,115 @@ for GBD are EHB from `Rhesus (Rh) disease`_, preterm birth,
...
+	* - deaths_c384
+	  - codcorrect
+	  - Count of deaths due to hemolytic diseases and other neonatal jaundice
+	  - 
+	* - population
+	  - Demography
```
Comment (NathanielBlairStahn) [inline docs/source/models/gbd2017/causes/neonatal_jaundice/index.rst]: Maybe let's just standardize and make everything lower case, i.e. "demography"?  I'll edit my table as well.

File: docs/source/models/gbd2017/causes/neonatal_jaundice/index.rst [hunk 9]
```diff
@@ -25,3 +27,115 @@ for GBD are EHB from `Rhesus (Rh) disease`_, preterm birth,
...
+	  - Mid-year population for given sex/age/year/location
+	  -
+	* - prevalence_s{id}
```
Comment (NathanielBlairStahn) [inline docs/source/models/gbd2017/causes/neonatal_jaundice/index.rst]: I went with prevalence_s{`sid`} in my table to unambiguously re-iterate that it's a sequela id, and I used the default Sphinx role ` ` to indicate a variable name.

**Pull request merged**